plugin-api/src/main/java/org/hyperledger/besu/plugin/data/Address.java

```diff
@@ -16,6 +16,8 @@
 
 import org.hyperledger.besu.plugin.Unstable;
 
-/** An interface for {@link BinaryData} that also represents an Ethereum account address. */
+import org.apache.tuweni.bytes.Bytes;
```

Comment:
It was a deliberate design decision to only expose `byte[]` and no other data library as part of the `plugin-api`.  Similarly the use of Quantity and UnformattedData to match their specified usages in the JSON-RPC apis.  Pleas undo all changes to this submodule.

Reply:
Will do.

Reply:
fixed now.